.github/workflows/demo_test.yml

```diff
@@ -871,3 +871,24 @@ jobs:
 
     - name: Run runtime boot instance
       run: docker exec ${{ github.job }} bash -c "cd /root/occlum/demos/runtime_boot/boot_instance && occlum run /bin/occlum_bash_test.sh"
+
+  Swtpm_test:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+
+    - uses: ./.github/workflows/composite_action/sim
+      with:
+        container-name: ${{ github.job }}
+        build-envs: 'OCCLUM_RELEASE_BUILD=1'
+
+    - name: download and build swtpm
+      run: docker exec ${{ github.job }} bash -c "cd /root/occlum/demos/swtpm; ./install_swtpm.sh"
+
+    - name: Run swtpm server
+      run: docker exec ${{ github.job }} bash -c "cd /root/occlum/demos/swtpm; SGX_MODE=SIM ./run_swtpm.sh"
```

Comment:
`run_swtpm.sh` will run as a server and this script will never return. You can take a look at the `Https_server_test` test in this file to use the background process to run this test.

Reply:
I updated it following the` Https_server_test`. Hope the CI will pass this time